bin/query-db-and-email

```diff
@@ -633,14 +652,15 @@ function getResponse (apiPath, qs) {
 }
 
 // TODO: clean this up?
+// TODO: rewrite as promises
 function getWithRetries(url, options, retries, callback) {
   if (typeof retries === 'function') {
     [callback, retries] = [retries, 2];
   }
   let retryCount = 0;
 
   function handleResult (error, response) {
-    if ((error || response.status >= 500) && retryCount <= retries) {
+    if ((error || response.statusCode >= 500) && retryCount <= retries) {
```

Comment:
Honestly this one change may have been a bigger deal than the caching! I think I was failing to actually retry in a number of cases :\